packages/admin-ui/ui/src/domain/orders/details/index.tsx

```diff
@@ -417,7 +427,8 @@ const OrderDetails = () => {
                   customActionable={
                     order.fulfillment_status !== "fulfilled" &&
                     order.status !== "canceled" &&
-                    order.fulfillment_status !== "shipped" && (
+                    order.fulfillment_status !== "shipped" &&
+                    anyItemsToFulfil && (
```

Comment:
suggestion: wouldn't `anyItemsToFulfill` overrule anything else, but canceled. I.e.:

```ts
                    order.status !== "canceled" &&
                    anyItemsToFulfil && (
```

Reply:
Yes, you're right, makes it more readable 👍